openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -104,6 +115,169 @@
+                "meeting_id": instance["id"],
+                "permissions": [
+                    "agenda_item.can_manage",
+                    "assignment.can_manage",
+                    "assignment.can_nominate_self",
+                    "list_of_speakers.can_be_speaker",
+                    "list_of_speakers.can_manage",
+                    "mediafile.can_manage",
+                    "meeting.can_see_frontpage",
+                    "meeting.can_see_history",
+                    "motion.can_manage",
+                    "projector.can_manage",
+                    "tag.can_manage",
+                    "user.can_manage",
+                    "user.can_change_own_password",
+                ],
+            }
+        elif CreateActionClass == GroupCreate and index == 5:
+            return {
+                "name": "Committees",
+                "meeting_id": instance["id"],
+                "permissions": [
+                    "agenda_item.can_see_internal",
+                    "assignment.can_see",
+                    "list_of_speakers.can_see",
+                    "mediafile.can_see",
+                    "meeting.can_see_frontpage",
+                    "motion.can_create",
+                    "motion.can_create_amendments",
+                    "motion.can_support",
+                    "projector.can_see",
+                    "user.can_see",
+                ],
+            }
+        elif CreateActionClass == ProjectorCreateAction and index == 6:
+            return {
+                "name": "Default projector",
+                "meeting_id": instance["id"],
+                "used_as_reference_projector_meeting_id": instance["id"],
+                "used_as_default_$_in_meeting_id": {
+                    name: instance["id"]
+                    for name in meeting_projector_default_object_list
+                },
+            }
+        raise RuntimeError(
+            f"Index {index} is not defined in this get_dependent_action_payload-method"
+        )
+
+    def validate_fields(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Check for guest_ids being in committee/member_ids or committee/manager_ids
+        """
+        instance = super().validate_fields(instance)
+        if instance.get("guest_ids"):
+            committee = self.datastore.get(
+                FullQualifiedId(Collection("committee"), instance["committee_id"]),
+                ["member_ids", "manager_ids"],
+            )
+            diff = (
+                set(cast(Iterable[Any], instance.get("guest_ids")))
+                - set(cast(Iterable[Any], committee.get("member_ids", ())))
+                - set(cast(Iterable[Any], committee.get("manager_ids", ())))
+            )
+            if diff:
+                raise ActionException(
+                    f"Guest-ids {diff} are not part of committee-member or manager_ids."
+                )
+        return instance
+
+
 @register_action_set("meeting")
```

Comment:
You could split this into seperate classes if you want - the action set is only useful for very simple actions.

Reply:
it's just the create